views/actions/src/components/EDAPlaceholder.vue

```diff
@@ -11,12 +15,29 @@ export default defineComponent({
             state: globalState
         };
     },
-    methods: {}
+    methods: {
+        executeCommand(command: string) {
+            vscode.commands.executeCommand(command);
```

Comment:
You can't execute a command from within the view. You need to send a message to the main extension, using `vscode.postMessage` from `views/actions/src/vscode`. This message then needs to be handled in `src/extension/webviews/actions.ts`. That is where you can execute the commands using `vscode.commands.executeCommand`.